documentation/wallet.adoc

```diff
@@ -0,0 +1,853 @@
+    }
+}
+
+user --> stationFront_listWebSites
+user --> stationFront_listModules
+user --> stationFront_useDApp
+user --> walletWebFront_createAccount
+user --> walletWebFront_importAccount
+user --> walletWebFront_listAccounts
+user --> walletWebFront_sendReceiveMassa
+user --> walletWebFront_checkMassaBalance
+user --> walletWebFront_importFungibleTokenAddress
+user --> walletWebFront_checkTokenBalance
+user --> walletWebFront_backupWallet
+user --> walletFront_enterConfirmPassword
+user --> walletFront_copyPastePrivateKey
+
+walletWebFront_createAccount                --> walletAPI_cRUDAccount
+walletWebFront_importAccount                --> walletAPI_cRUDAccount
+walletWebFront_listAccounts                 --> walletAPI_cRUDAccount
+walletWebFront_sendReceiveMassa             --> walletAPI_sendMassa
+walletWebFront_checkMassaBalance            --> walletAPI_cRUDAccount
+walletWebFront_importFungibleTokenAddress   --> walletAPI_cRUDAsset
+walletWebFront_checkTokenBalance            --> walletAPI_cRUDAsset
+walletWebFront_backupWallet                 --> walletAPI_backupAccount
+
+stationFront_useDApp --> walletAPI_serveWebSite
+stationFront_useDApp --> walletAPI_cRUDAccount
+stationFront_useDApp --> walletAPI_sendMassa
+stationFront_useDApp --> walletAPI_buySellRolls
+stationFront_useDApp --> walletAPI_signOperation
+stationFront_useDApp --> walletAPI_signMessage
+stationFront_useDApp --> walletAPI_backupAccount
+stationFront_useDApp --> walletAPI_cRUDAsset
+@enduml
+----
+
+=== Sequence Diagram
+
+This section provides sequence diagrams for main functionalities of each 
+Massa Wallet component.
+
+==== Sign Operation
+
+This sequence diagrams illustrates signing a Transaction operation. It applies to all other operation types: Roll buy and Roll sell, CallSC and ExecuteSC.
+
+[plantuml, format="svg", id="sign-operation", width="100%", role="center-image", options="interactive"]
+----
+@startuml
+
+skinparam FontName Poppins
+
+skinparam sequence {
+    ArrowColor #69679D
+    LifeLineBorderColor #69679D
+    LifeLineBackgroundColor #F5F5FF
+
+    ParticipantBorderColor #69679D
+    ParticipantBackgroundColor #F5F5FF
+    ParticipantFontName Poppins
+    ParticipantFontSize 16
+    ParticipantFontColor #000000
+}
+
+participant "User" as user
+participant "dApp or Wallet" as dapp
+participant "Massa Station" as station
+participant "Massa Wallet API" as wallet_api
+participant "Massa Wallet Core" as wallet_core
+participant "Massa Wallet Front" as wallet_embedded_front
+
+user -> dapp ++ : Click "Send Coins"
+    dapp -> station ++: Request //Sign Operation//
+        station -> wallet_api ++: Request //Sign Operation//
+        wallet_api -> wallet_api: Execute //Validate Arguments//
+            wallet_api -> wallet_core ++: Execute //Sign Operation//
+            wallet_core -> wallet_core: Fetch //Account Data//
+                wallet_api -> wallet_front ++: Fetch //Operation details
```

Comment:
I don't understand this message: How the wallet_api can fetch detail from the front ? details are sent as arguments of the _Sign Operation_ request, aren't they? Why is it opening a new life line?